openmoc/plotter.py

```diff
@@ -54,12 +59,17 @@
 # @endcode
 #
 # @param track_generator the TrackGenerator which has generated Tracks
-def plot_tracks(track_generator):
-
-    global subdirectory
+# @param get_figure whether or not to return the Matplotlib figure
+def plot_tracks(track_generator, get_figure=False):
 
+    global subdirectory, font
     directory = openmoc.get_output_directory() + subdirectory
 
+    # Ensure that normal settings are used even if called from ipython
+    curr_rc = dict(matplotlib.rcParams)
+    matplotlib.rcParams.update(matplotlib.rcParamsDefault)
+    plt.rc('font', **font)
```

Comment:
Ok. How about we save the ``rcParams`` we want to a class or global variable? That way we would just need to set the parameters using that variable.